app/components/impagination-dataset.js

```diff
@@ -1 +1,116 @@
-export { default } from 'ember-impagination/components/impagination-dataset';
```

Comment:
I was able to move everything back into the `addon` directory. However, I'm doing this now inside `app/components/impagination-dataset.js`

``` javascript
import Dataset from 'npm:impagination';
import EmberImpagination from 'ember-impagination/components/impagination-dataset';
export default EmberImpagination.extend({ Dataset });
```

Thoughts, comments, concerns?